woocommerce/woocommerce-android
# AI Product description: Powered by AI link

Fixes #9289, a subtask of #9214.

![image](https://github.com/woocommerce/woocommerce-android/assets/1522856/49e2a37a-fd96-42e9-8aa7-744ded50371d)

The PR adds a `Powered by AI` link underneath the `Write with AI` button. It opens the Automattic AI guidelines web page.

**To test:**
1. Go to Products -> select a product
2. Notice there's a new link underneath the `Write with AI` button
3. Tap on it and verify the `https://automattic.com/ai-guidelines/` web site is opened

**Note:** Please merge #9293 first.
